opm/grid/cpgrid/CpGridData.hpp

```diff
@@ -77,6 +77,7 @@
 namespace Opm
 {
 class EclipseState;
+class NNC;
```

Comment:
Do we need this?

Reply:
We needed it in CpGrid.hpp, and I added it also here assuming we also need it here (Opm::NNC is the type of one of the arguments of the function defined in the new test file, and not having it declared in CpGrid.hpp was generating errors.)

Reply:
As we discussed, this is related to (the many) forward and **friend** declarations of free functions defined in test files, due to the need of accessing certain private member(s) of the class. It will be solved/modified in a separate PR.

Reply:
I see. Thanks for the explanation,